test/verify/check-packagekit

```diff
@@ -997,6 +997,8 @@ class TestUpdatesSubscriptions(PackageCase):
 
         # make sure that rhsm skips certificate checks for the server
         self.sed_file("s/insecure = 0/insecure = 1/g", "/etc/rhsm/rhsm.conf")
+        # enable subscriptions in dnf
+        self.sed_file("/enabled/ s/0/1/", "/etc/yum/pluginconf.d/subscription-manager.conf")
```

Comment:
That's a weird regexp but I am sure you know what you doing. But out of curiosity what's the difference between this and 's/enabled=0/enabled=1'?

Reply:
Not that much difference, just that one does not need to jump through hoops to specify the whole line. e.g. with arbitrary spaces in between, or which separator, etc. This says "in every line that contains "enabled", replace "0" with "1".

Maybe I should be a litte more precise though and say `/^enabled/`. I can re-post.

Reply:
I think it's fine. After all it's a test.

Reply:
I made it stricter and in the form you suggested, if that's easier to read.

Reply:
It was more a question than suggestion :) but its' fine either way for me.